core/domain/html_validation_service_test.py

```diff
@@ -979,46 +981,64 @@ def test_validate_customization_args_in_tag(self):
 
         self.assertEqual(actual_output, expected_output)
 
-    def test_add_dimensions_to_noninteractive_image_tag(self):
+    def test_add_dimensions_to_image_tags(self):
         test_cases = [{
             'html_content': (
                 '<p><oppia-noninteractive-image filepath-with-value="&amp;quot;'
-                'abc.png&amp;quot;"></oppia-noninteractive-image>Hello this'
+                'abc1.png&amp;quot;"></oppia-noninteractive-image>Hello this'
                 ' is test case to check that dimensions are added to the oppia'
                 ' noninteractive image tags.</p>'
             ),
             'expected_output': (
                 u'<p><oppia-noninteractive-image filepath-with-value='
-                '"&amp;quot;abc_height_32_width_32.png&amp;'
+                '"&amp;quot;abc1_height_32_width_32.png&amp;'
                 'quot;"></oppia-noninteractive-image>Hello this is test case'
                 ' to check that dimensions are added to the oppia '
                 'noninteractive image tags.</p>'
             )
         }, {
             'html_content': (
                 '<p><oppia-noninteractive-image filepath-with-value="&amp;quot;'
-                'abc.png&amp;quot;"></oppia-noninteractive-image>Hello this'
+                'abc2.png&amp;quot;"></oppia-noninteractive-image>Hello this'
                 ' is test case to check that dimensions are added to the oppia'
-                ' noninteractive image tags.</p>'
+                ' noninteractive image tags.<oppia-noninteractive-image '
+                'filepath-with-value="&amp;quot;abc3.png&amp;quot;">'
+                '</oppia-noninteractive-image></p>'
             ),
             'expected_output': (
-                u'<p><oppia-noninteractive-image filepath-with-value='
-                '"&amp;quot;abc_height_32_width_32.png&amp;'
-                'quot;"></oppia-noninteractive-image>Hello this is test case'
-                ' to check that dimensions are added to the oppia '
-                'noninteractive image tags.</p>'
+                u'<p><oppia-noninteractive-image filepath-with-value="'
+                '&amp;quot;abc2_height_32_width_32.png&amp;quot;">'
+                '</oppia-noninteractive-image>Hello this is test case '
+                'to check that dimensions are added to the oppia'
+                ' noninteractive image tags.<oppia-noninteractive-image '
+                'filepath-with-value="&amp;quot;abc3_height_32_width_32.png'
+                '&amp;quot;"></oppia-noninteractive-image></p>'
+            )
+        }, {
+            'html_content': (
+                '<p>Hey</p>'
```

Comment:
Put some content here to indicate the purpose of the test. Maybe something like "Hey this is a test case with no images."

Reply:
Done